src/Notification/NotificationActionButton.svelte

```diff
@@ -1,5 +1,7 @@
 <script>
   import Button from "../Button/Button.svelte";
+  
+  /** @extends {"../Button/Button.svelte"} ButtonProps */
```

Comment:
This is great – thanks for the fix.

For consistency, could the comment be moved to [top of the script block](https://github.com/carbon-design-system/carbon-components-svelte/blob/ab5eff5205587a88b1682ec254e17184c991c010/src/Accordion/Accordion.svelte#L2)?

Reply:
moved!